ui/app/views/service/status.html

```diff
@@ -187,29 +187,49 @@ <h4 class="modal-title" id="myModalLabel">Are you sure?</h4>
         </div>
     </div>
 
-    <div class="modal fade" id="select-container" tabindex="-1" role="dialog" aria-hidden="true" style="display: none;">
+    <div class="modal fade" id="agree-to-open-exec" tabindex="-1" role="dialog" aria-hidden="true" style="display: none;">
         <div class="modal-dialog">
             <div class="modal-content">
                 <div class="modal-header">
                     <button type="button" class="close" data-dismiss="modal" aria-hidden="true">×</button>
-                    <h4 class="modal-title">Select a container</h4>
+                    <h4 class="modal-title">Are you sure you wanna do this?</h4>
                 </div>
                 <div class="modal-body">
-                    <div class="row">
-                        <div class="row text-center">
-                            <button type="button" class="btn btn-outline btn-primary btn-lg apollo-version-selector"
-                                    ng-repeat="container in selectedPodStatus.containers" ng-click="startWebSocket(container)" data-toggle="modal" data-target="#select-container,#live-session">
-                                {{container}}
-                            </button>
-                        </div>
-                    </div>
+                    Watch out! You are going to open a live terminal to the container! <br>
```

Comment:
This will automatically grant you exec permissions to a running live container, this session will be documented and monitored.